### PR TITLE
Update README examples to work with JuMP v0.19.0

### PR DESCRIPTION
**Pull request by r-barnes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ To use these settings you can either pass them as keyword arguments to `setup` (
 # Direct
 my_prob = ECOS.setup(n, m, ..., c, h, b; maxit=10, feastol=1e-5)
 # MathProgBase (with JuMP)
```
Comment (blegat) [inline README.md]: Replace MathProgBase by MathOptInterface

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ To use these settings you can either pass them as keyword arguments to `setup` (
...
-m = Model(solver=ECOS.ECOSSolver(maxit=10, feastol=1e-5))
+m = Model(with_optimizer(ECOS.Optimizer, maxit=10, feastol=1e-5))
```
Comment (blegat) [inline README.md]: It should be named `model` according to the JuMP style guide

**Comment by r-barnes** [comment]
It was called `m` previously and I didn't want to break the more localized ECOS convention.

File: README.md (hunk 2 above)
Comment (r-barnes) [reply README.md]: It was called `m` previously and I didn't want to break the more localized ECOS convention.

Comment (odow) [reply README.md]: Feel free to rename everything so that it conforms to the style guide http://www.juliaopt.org/JuMP.jl/v0.19.0/style/

Comment (blegat) [reply README.md]: `m` is not related to ECOS, it is how we used to name the model everywhere but it should now be `model` :)

Comment (r-barnes) [reply README.md]: Updated to meet stylistic requests.

**Pull request merged**